goty-server/src/main/kotlin/com/aleinin/goty/configuration/goty/GOTYConfigurationService.kt

```diff
@@ -0,0 +1,21 @@
+package com.aleinin.goty.configuration.goty
+
+import com.aleinin.goty.configuration.DefaultGOTYConfiguration
+import org.springframework.stereotype.Service
+import org.springframework.validation.annotation.Validated
+
+@Service
+class GOTYConfigurationService(
+    private val GOTYConfigurationRepository: GOTYConfigurationRepository,
+    @Validated private val defaultConfiguration: DefaultGOTYConfiguration
+) {
+
+    fun getConfiguration(): GOTYConfiguration = GOTYConfigurationRepository
+        .findConfiguration()
+        .orElseGet { loadDefaultValues() }
+
+    fun replaceConfiguration(request: GOTYConfiguration) = GOTYConfigurationRepository.replaceConfiguration(request)
+
+    private fun loadDefaultValues() = GOTYConfigurationRepository.replaceConfiguration(defaultConfiguration)
```

Comment:
Why is a function to load properties calling a replace function?

Reply:
The intention was to save the defaults to the database. In hindsight, this isn't really necessary and this functionality should be changed.

Reply:
This was removed. Functionality now:

1. Attempts to load from the DB, returning it if available
2. Returns the default config if DB misses